holoviews/ipython/__init__.py

```diff
@@ -246,6 +249,14 @@ def load_hvjs(cls, logo=False, bokeh_logo=False, mpl_logo=False, plotly_logo=Fal
                 MIME_TYPES['jlab-hv-load'] : widgetjs
             })
 
+    @classmethod
+    def _process_comm_msg(cls, msg):
```

Comment:
Maybe this method could have a more specific name like ``_delete_plot_by_id``?

Reply:
I purposefully kept this general, I could imagine sending other types of messages, e.g. to shut down bokeh server sessions launched inside the notebook and if we ever find it's needed we could actually distinguish between JupyterLab and classic notebooks using a message.

Reply:
Sure. Personally I would use a specific name for the method while it *is* specific and rename it once it does handle the more general case. I don't mind leaving it as it is though...